governance/votes/Executive vote - June 7, 2023.md

```diff
@@ -0,0 +1,215 @@
+- Increase the RWA012-A Debt Ceiling by 50 million DAI from 30 million DAI to **80 million DAI**.
+
+To facilitate these changes, the [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) for RWA010-A, RWA011-A, RWA012-A, and RWA013-A will be updated to **QmY382BPa5UQfmpTfi6KhjqQHtqq1fFFg2owBfsD2LKmYU**, the agreement details may be [viewed here](https://gateway.pinata.cloud/ipfs/QmY382BPa5UQfmpTfi6KhjqQHtqq1fFFg2owBfsD2LKmYU).
+
+### Stability Scope Defined Parameter Changes
+
+As per the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104), the following parameter changes will be made, if this executive proposal passes:
+
+#### [Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) Increase
+- Increase the DSR by 2.49% from 1% to **3.49%**.
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Increases
+- Increase the ETH-A Stability Fee by 1.99% from 1.75% to **3.74%**.
+- Increase the ETH-B Stability Fee by 0.99% from 3.25% to **4.24%**.
+- Increase the ETH-C Stability Fee by 2.49% from 1% to **3.49%**.
+- Increase the WSTETH-A Stability Fee by 1.99% from 1.75% to **3.74%**.
+- Increase the WSTETH-B Stability Fee by 2.49% from 1% to **3.49%**.
+
+### Non-Scope Defined Paramater Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmQXhS3Z), the following parameter changes will be made, if this executive proposal passes:
+
+#### [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam) Increase
+- Increase the RETH-A Maximum Debt Ceiling by 30 million DAI from 20 million DAI to **50 million DAI**.
+
+#### [Target Availible Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) Increase
+- Increase the RETH-A Target Available Debt by 2 million DAI from 3 million DAI to **5 million DAI**.
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Increases
+- Increase the RETH-A Stability Fee by 2.99% from 0.75% to **3.74%**.
+- Increase the CRVV1ETHSTETH-A Stability Fee by 2.49% from 1.75% to **4.24%**.
+- Increase the WBTC-A Stability Fee by 0.9% from 4.90% to **5.80%**.
+- Increase the WBTC-B Stability Fee by 1.4% from 4.90% to **6.30%**.
+- Increase the WBTC-C Stability Fee by 0.65% from 4.90% to **5.55%**.
+
+### USDP PSM [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) Reduction
+
+As per [this poll](https://vote.makerdao.com/polling/QmQYSLHH), the following change to the USDP PSM Debt Ceiling will be made, if this executive proposal passes:
+- Reduce the USDP PSM (technical name PSM-PAX-A) Debt Ceiling by 500 million DAI from 500 million DAI to **0**.
+
+*Technical implementation note:* to achieve this, the USDP PSM will be removed from the Autoline (DC-IAM) and have its Debt Ceiling set to 0.
```

Comment:
Probably unnecessary inclusion imo, but not a big deal.

Reply:
always find this difficult to balance, try to explain the underlying technical changes for those checking the code, but does run the risk of over-complication